### DISCUSSION TITLE
HomeKit SimpliSafe Camera not showing live veiew

### DISCUSSION THREAD

**Discussion by sjaweed** [homebridge-simplisafe3/homebridge-simplisafe3#382] [Q&A]
Good day,

I was hoping to see I can get help. I finally setup Homebridge and installed your plugin with UI and it went well.
I had an issue with seeing anything but that was an account issue which since has been fixed

I now see my devices but don’t see live view in HomeKit it just spins and dies. Any advice ?

**Comment by shamoon** [comment]
Glad it's gotten setup, think I need more specifics with the camera. E.g. what type of camera, installation specifics and if any log output.

**Comment by sjaweed** [comment]
actually, i am making an edit, i ended up completely removing and reinstalling plugin, and it works well.

this is using hombridge for windows. i will migrate to a linux sever this weekend.

i truly appreciate your efforts.

I wan tot add,
I want to support this project, I would like to send you the devices you don't have so you can build on it. please tell me if that is something you wish to do, I read on another thread that you don't have the outside camera and such so i am happy to sent you a couple or other sensors or both let me know.

**Reply by shamoon** [reply]
Glad it’s working. Yes logs looked like an ffmpeg issue which isn’t the best on windows.

As for the camera, appreciate the offer but when I last looked into it I had major concerns about the speed of using those via homebridge and as of yet I don’t think anyone (myself included) has successfully reverse-engineered the stream from them as it’s a completely different than the current cameras.

see #240 for the whole discussion of that
